rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -242,8 +250,51 @@ Open the xref:processoperations-financeanalysis.adoc[*Finance Analysis* view] to
 
 Once it's ready, transition the project to the next phase of its life-cycle. You can always transition back to change the project's design or implementation.
 
+In the **Design** phase, you can do the following:
```

Comment:
What does the double asterisk do?

Reply:
If you want to emphasize the phase, suggest using the definition list.

Reply:
Changes to definition list.